configs/vail.nrel-op.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://vail-openpath.nrel.gov/api/",
```

Comment:
@shankari can you please confirm which `connectUrl` we want for Vail? Thanks!

Reply:
The base URL should be
https://e-bikes-for-essentials-openpath.nrel.gov

Note that you also need to rename the spec file appropriately